src/utils/formatDatasetLanguage.ts

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+import { iso6393 } from 'iso-639-3/iso6393';
+
+export default function formatDatasetLanguage(language: string) {
```

Comment:
This should not be needed, as the URI labels will common from the backend.

Reply:
@brunopacheco1 Right now they are not coming. Do we plan to fix this?

<img width="1005" alt="Screenshot 2024-02-26 at 09 24 54" src="https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/assets/1210046/84b7aa2a-a43c-4f87-8bdc-1d4ebdc02f28">